### PR TITLE
fix/PLF-7241:Wrong width of notification bar

### PR DESCRIPTION
**Pull request by mbenrhouma**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less [hunk 1]
```diff
@@ -455,6 +455,7 @@
 		display:none;
 	}
 	.dropdown-menu {
+        width : 315px;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less]: Does it work fine in tablet and phone screens ?

Comment (mbenrhouma) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less]: Of course, it work fine

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less [hunk 2]
```diff
@@ -533,7 +534,7 @@
     }
 	.displayItems {
 		min-width: 300px;
-		max-width: 300px;
+		max-width: 313px;
```
Comment (alaameddeb) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less]: you should move max-width and min-width to child element .media with : 
.media {
    min-width: 300px;
    max-width: 100%;
}

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less [hunk 3]
```diff
@@ -455,6 +455,7 @@
 		display:none;
 	}
 	.dropdown-menu {
+        width : -moz-max-content;
```
Comment (alaameddeb) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less]: useless property

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less (hunk 2 above)
Comment (fmaatoug) [reply platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less]: ok agreed

File: platform-ui-skin/src/main/webapp/skin/bootstrap/less/media.less [hunk 4]
```diff
@@ -18,6 +18,8 @@
 .media,
 .media .media {
   margin-top: 15px;
+  min-width: 300px;
+  max-width: 100%;
```
Comment (boubaker) [inline platform-ui-skin/src/main/webapp/skin/bootstrap/less/media.less]: This change will affect the max width on "All Notifications" screen.
Please move this class change to file
platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less
and add it under .displayItems, [here](https://github.com/exodev/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less#L534):

```
.displayItems {
...
  .media {
    min-width: 300px;
    max-width: 100%;
  }
...
}
```

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less (hunk 3 above)
Comment (boubaker) [reply platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less]: Please delete this newly added property.

File: platform-ui-skin/src/main/webapp/skin/bootstrap/less/media.less (hunk 4 above)
Comment (fmaatoug) [reply platform-ui-skin/src/main/webapp/skin/bootstrap/less/media.less]: done

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less (hunk 3 above)
Comment (fmaatoug) [reply platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less]: done

**Pull request merged**